PagerDuty partner docs (#5)
relay-integrations/relay-pagerduty: parter.md

# PagerDuty + Relay Integration Benefits
    
* Respond to PagerDuty incidents with Relay's event-driven workflow engine.
* Track details of triggered events in ticketing systems such as Jira. 
* Submit incident acknowledgements and changes from external sources such as Slack.
* Codify and automate incident response process saving valuable time and resources.
* Centralize audit trails of actions taken when responding to a production incident.

# How it Works
* Incidents from PagerDuty can trigger Relay workflow runs to automatically address infrastructure needs and coordinate staff responses using the catalog of Relay providers. Whether restarting ec2 instances, logging incident details in Jira, setting up war rooms in Slack, or gathering status information from Kubernetes, Relay has you covered.

> * Relay workflow failures can be submitted to PagerDuty to trigger incidents as well, reducing response time to failed deployments or upgrades.

Reviewer: ```suggestion
* Relay can submit workflow failures to PagerDuty to trigger incidents as well, reducing response time to failed deployments or upgrades.
```

> * PagerDuty incidents may be acknowledged and resolved using the Relay PagerDuty integration as part of a workflow run.

Reviewer: ```suggestion
* The Relay PagerDuty integration can acknowledge and resolve PagerDuty incidents as part of a workflow run.
```

> * Similarly, PagerDuty change events detailing actions taken to address the incident may also be logged in PagerDuty by Relay workflow runs.

Reviewer: ```suggestion
* Similarly, Relay workflow runs can log PagerDuty change events detailing actions taken to address an incident.
```

   * **If you are creating a new service for your integration**: Please read PagerDuty's documentation in section [Services and Integrations](https://support.pagerduty.com/docs/services-and-integrations) and follow the steps outlined in the [Create a New Service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting Relay as the **Integration Type** in step 4. Continue with the [In  Relay](#in-relay)  section (below) once you have finished these steps.
3. Enter an **Integration Name** such as `Relay` or `relay-by-puppet` and select Relay from the Integration Type menu.
4. Click the **Add Integration** button to save your new integration. You will be redirected to the Integrations tab for your service.
5. An **Integration Key** will be generated on this screen. Keep this key saved in a safe place, as it will be used when you configure the integration with Relay in the next section.
6. Continue with the [In Relay](#in-relay) section below.
![](media/integration-key.png)

### Integrating Relay Triggers With a PagerDuty Service

> 1. From the **Configuration** menu, select **Services**.

Reviewer: The first step here should be to add the workflow to relay and expand the settings tab - otherwise they won't know where the webhook URL came from.
